python-packages/contract_wrappers/test/test_erc20_wrapper.py

```diff
@@ -38,8 +38,8 @@ def test_erc20_wrapper__balance_of(
     weth_instance.functions.deposit().transact(
         {"from": accounts[1], "value": expected_difference}
     )
-    acc1_weth_balance = erc20_wrapper.balance_of(weth_address, accounts[0])
-    acc2_weth_balance = erc20_wrapper.balance_of(weth_address, accounts[1])
+    acc1_weth_balance = erc20_wrapper.balanceOf(weth_address, accounts[0])
+    acc2_weth_balance = erc20_wrapper.balanceOf(weth_address, accounts[1])
```

Comment:
Is underscores the idiomatic python style for method names? If so, we might want to convert camel-case to underscores.